sfge/src/main/java/com/salesforce/graph/ops/MethodUtil.java

```diff
@@ -257,6 +260,21 @@ public static List<MethodVertex> getGlobalMethods(
                                         __.not(__.has(Schema.IS_STANDARD, true)))));
     }
 
+    public static List<MethodVertex> getInboundEmailHandlerMethods(
+            GraphTraversalSource g, List<String> targetFiles) {
+        return SFVertexFactory.loadVertices(
```

Comment:
The modular traversal added in `TraversalUtil` makes the bespoke parts of this traversal very straightforward.